docker/docker-compose.yml

```diff
@@ -33,8 +35,12 @@ services:
 
   nodejs:
     image: fidals/se-nodejs:dev
-    volumes_from:
-      - app
+    volumes:
+      - $FRONT_BUILD_DIR
+      # Volumes for refarm-site's front development
+      #- ../gulpfile.babel.js:/usr/app/src_front/gulpfile.babel.js
```

Comment:
Perhaps we should mount these volumes by default. It seems more convenient, because docker-compose.yml is used to for development purpose

Reply:
@artemiy312 , sorry, missed this comment.

I'll enable it buy default